include/nanorange/algorithm/stable_partition.hpp

```diff
@@ -95,37 +98,40 @@ struct stable_partition_fn {
         }
 
         const I first_false = (m1 == first) ? first :
-                impl_unbuffered(first, m1, len_half, pred, proj);
+                impl_unbuffered(first, m1, len_half, pred, proj).end();
```

Comment:
Trailing `.end()` looks ugly and is easy to miss. I'm not sure how I feel about it.

Reply:
This algorithm is quite messy anyway, don't worry about it :)

Reply:
If it works for you... \*shrug\* Good enough for me. :)